### PR TITLE
Update to Node v18

### PR DESCRIPTION
**Pull request by irisfaraway**
https://eaflood.atlassian.net/browse/IWTF-3729

### CODE REVIEW

**Comment by irisfaraway** [comment]
This PR also includes a lot of automated linting tidy-ups as a result of updating Prettier and our pre-commit checks.

File: package.json [hunk 1]
```diff
@@ -21,20 +21,20 @@
...
+    "docker:run-acceptance-tests": "docker run --env-file docker/env/frontend_acceptance_tests.env rod_licensing/frontend_acceptance_tests:latest",
+    "prepare": "husky install",
+    "pre-commit": "lint-staged"
   },
   "devDependencies": {
-    "eslint-plugin-chai-friendly": "^0.6.0",
-    "husky": "^4.3.0",
-    "lerna": "^3.22.1",
-    "lerna-update-wizard": "^0.17.8",
-    "prettier-standard": "^16.4.1",
```
Comment (jaucourt) [inline package.json:36]: Not sure why prettier's been removed?

File: package.json [hunk 2]
```diff
@@ -21,20 +21,20 @@
     "url": "git://github.com/DEFRA/rod-licensing-tests.git"
   },
   "scripts": {
-    "postinstall": "npm run lerna:bootstrap",
```
Comment (jaucourt) [inline package.json:24]: Why doesn't this need to run anymore?

**Review by jaucourt — COMMENTED** [review]
I can see there's a lot of work and research gone into this, I think I'd appreciate a walk through before we merge this

File: package.json (hunk 1 above)
Comment (jaucourt) [reply package.json:36]: Nevermind, I see now

**Review by jaucourt — APPROVED** [review]

**Pull request merged**